js/validation/validationSignIn.js

```diff
@@ -6,49 +6,39 @@ const validationSignIn = () => {
     '.error-message__password'
   )
 
-  if (email.value === '') {
-    email.classList.add('form__input_error')
-    errorMessageEmail.innerHTML = 'Please enter your email.'
-    errorMessageEmail.classList.add('form__text_error')
-    return false
-  } else {
-    email.classList.remove('form__input_error')
-    errorMessageEmail.innerHTML = ''
+  const ifFieldValid = (target, message, reg) => {
```

Comment:
Move this function into separete file, and use it in SignIn SugnUp anywhere you want in future.